### PR TITLE
Mac: Add Universal 2 support

### PR DESCRIPTION
**Pull request by MaddTheSane**
Target and build x86_64 and Arm64.
Note: you will need Xcode 12.2 which will probably be released when Big Sur/Apple Silicon Macs are released (it's currently in beta).

### CODE REVIEW

**Comment by hrydgard** [comment]
I'm thinking maybe we hold off until it's out of beta, to not have to commit multiple versions here? Or maybe it's stable enough already?

**Comment by MaddTheSane** [comment]
Agreed. Xcode 12 is out of beta, but it doesn’t have the macOS 11 SDK yet.

File: mac-build.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (orbea) [inline mac-build.sh:1]: This should not be hard coded to bash.

Comment (MaddTheSane) [reply mac-build.sh:1]: My changes were based off of the iOS build script, which also uses a hard-coded Bash. I'll change it to just plain sh.

Comment (orbea) [reply mac-build.sh]: I only noticed the script that was being replaced was `/bin/sh`. I believe osx is now using zsh by default?

Comment (MaddTheSane) [reply mac-build.sh]: As of 10.15 (Catalina), yes.

**Comment by hrydgard** [comment]
@MaddTheSane I'm looking into supporting the M1 now. Would you like to rebuild this with the latest XCode?

**Comment by MaddTheSane** [comment]
Will do

**Comment by hrydgard** [comment]
Thanks!

It works fine on an M1 Air. (PPSSPP itself needs a few minor touches, plus still struggling a little with SDL2 pkg-config but hackily got it to work)

**Comment by MaddTheSane** [comment]
You're welcome.

**Pull request merged**